data/json/furniture_and_terrain/furniture-graves.json

```diff
@@ -95,5 +95,35 @@
       "sound_fail": "thunk!",
       "items": [ { "item": "rock", "count": [ 18, 30 ] } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_battlefield_cross",
+    "name": "battlefield cross",
+    "description": "A makeshift memorial marker, made of a service rifle - buried in the dirt, or nestled in a pair of boots - with a helmet perched on the stock.  A small way to honor the dead.",
+    "symbol": "(",
+    "color": "dark_gray",
+    "move_cost_mod": 0,
+    "coverage": 20,
+    "required_str": -1,
+    "flags": [ "EASY_DECONSTRUCT", "TRANSPARENT", "MOUNTABLE" ],
+    "deconstruct": {
+      "items": [
+        { "item": "m4_carbine", "count": 1, "charges": 0 },
+        { "item": "helmet_army", "count": 1, "damage": [ 0, 3 ] },
+        { "item": "boots_combat", "count": [ 0, 1 ], "damage": [ 0, 3 ] }
+      ]
+    },
+    "bash": {
+      "str_min": 1,
+      "str_max": 4,
+      "sound": "thump.",
+      "sound_fail": "whack.",
+      "items": [
+        { "item": "m4_carbine", "count": 1, "charges": 0 },
```

Comment:
Since it's a bash result, maybe add chance for some damage?

Reply:
I'd thought about it, but I decided against. Guns are pretty durable, and since this is just a simple marker, I figured you could easily just kick it over to bring it down without any real damage to the components. The helmet and boots are damaged primarily as a balance thing (the M4 has no magazine or ammo, which is its particular balance bit.)